### PR TITLE
Add phpdoc documentation

### PR DESCRIPTION
**Pull request by Paxa**

### CODE REVIEW

File: Veritrans/Config.php [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
+   * @static
+   */
   public static $serverKey;
+  /**
+   * Your merchant's client key
+   * @static
+   */
   public static $client_key;
+  /**
+   * This should be 2
```
Comment (rafeequl) [inline Veritrans/Config.php]: give the explanation why 2 ?

File: Veritrans/Config.php [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
+   * @static
+   */
   public static $apiVersion = 2;
+  /**
+   * Make it true to integrate with production
```
Comment (rafeequl) [inline Veritrans/Config.php]: true for production
false for sandbox mode

File: Veritrans/Config.php [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
+   * @static
+   */
   public static $isProduction = false;
+  /**
+   * Enable 3D Secure by default
```
Comment (rafeequl) [inline Veritrans/Config.php]: enable 3ds by default but it set to false ?

Comment (Paxa) [inline Veritrans/Config.php]: This key is to enable/disable 3d secure by default. Have better idea about description?

File: Veritrans/Config.php (hunk 1 above)
Comment (Paxa) [inline Veritrans/Config.php]: It's never used in code, but I think it suppose to change 'https://api.veritrans.co.id/v2' -> 'https://api.veritrans.co.id/v3'. I think better remove it

**Comment by Paxa** [comment]
Should we merge it or delete it?

File: Veritrans/Config.php [hunk 4]
```diff
@@ -1,17 +1,43 @@
...
+   * Your merchant's server key
+   * @static
+   */
   public static $serverKey;
+  /**
+   * Your merchant's client key
+   * @static
+   */
   public static $client_key;
-  public static $apiVersion = 2;
```
Comment (rafeequl) [inline Veritrans/Config.php]: expected to remove this line ?

**Comment by rafeequl** [comment]
it should be included in previous pull request?

**Pull request closed without merging**